extensions/podman/packages/extension/src/extension.spec.ts

```diff
@@ -1795,7 +1795,8 @@ describe('initCheckAndRegisterUpdate', () => {
       stdout: 'podman version 4.0',
     } as unknown as extensionApi.RunResult);
     // call the updateVersion
-    await func();
+    expect(func).not.toBeUndefined();
+    await func!('v1');
```

Comment:
please avoid any ! usage, use rather ?

also why `expect(func).not.toBeUndefined();` vs `expect(func).toBeDefined()`